src/static/css/pages/beta.less

```diff
@@ -161,10 +157,12 @@
 .progress-charts_chart {
     width: 100px;
     height: 100px;
+    margin-right: 1em;
+    float: left;
 
-    .respond-to-max(479px, {
-        margin-right: 1em;
-        float: left;
+    .respond-to-min(@tablet-min, {
```

Comment:
The idea was to try to move towards being more mobile first, but with some distance, I think you're right and it's better to be DRY-er.